pheweb/serve/data_access/db.py

```diff
@@ -886,9 +896,11 @@ def __init__(self, manifest):
                         raise Exception("All required headers not found in external result file:" + line[file_idx] + ". Required fields: " +
                          ",".join(self.FILE_REQ_HEADERS))
 
+                    self.chromprefix = "chr" if f.readline().startswith("chr") else ""
+
                     self.results[ line[name_idx] ] = ExternalFileResultDao.ResRecord( line[name_idx], line[ncase_idx], line[ncontrol_idx],
-                        line[file_idx] ,header.index("achr38"), header.index("apos38"), header.index("REF") ,header.index("ALT") ,
-                        header.index("beta"), header.index("pval") )
+                        line[file_idx] ,header.index(self.column_names["chr"]), header.index(self.column_names["pos"]), header.index(self.column_names["ref"]) ,header.index(self.column_names["alt"]) ,
```

Comment:
not a crucial one but could create dict of colname->index to make code shorter. Also does not need to scan through all columns although in single file there aren't many anyway.